Jenkinsfile.talend

```diff
@@ -1,6 +1,6 @@
 def GIT_URL = 'https://github.com/Talend/apache-camel.git'
 def DEFAULT_BRANCH = 'tesb/camel-2.23.1.x'
-def DEFAULT_MVN_MODULES = 'org.apache.camel:camel-mqtt'
+def DEFAULT_MVN_MODULES = 'org.apache.camel:camel-saxon'
```

Comment:
Need to keep the existing DEFAULT_MVN_MODULES and add yours.
Something like
```
def DEFAULT_MVN_MODULES = 'org.apache.camel:camel-mqtt,org.apache.camel:camel-saxon'
```

Reply:
Jenkins build file is corrected. A local test build has produced the right versions of the artifacts.